edx_dl/edx_dl.py

```diff
@@ -1008,11 +1012,13 @@ def main():
     # Parse the sections and build the selections dict filtered by sections
     if args.platform == 'edx':
         all_selections = {selected_course:
-                          get_available_sections(selected_course.url.replace('info', 'course'), headers)
+                          get_available_sections(
+                              selected_course.url.replace('info', 'course'), headers)
```

Comment:
You can leave the first argument on the first line and leave the 2nd on a new line.